shell/platform/tizen/channels/localization_channel.cc

```diff
@@ -21,30 +21,50 @@ LocalizationChannel::LocalizationChannel(FLUTTER_API_SYMBOL(FlutterEngine)
 LocalizationChannel::~LocalizationChannel() {}
 
 void LocalizationChannel::SendLocales() {
+  const char* defualt_locale = nullptr;
+  FlutterLocale* flutter_locale = nullptr;
+  std::vector<FlutterLocale*> flutter_locales;
+
+  int ret = i18n_ulocale_set_default(getenv("LC_TIME"));
+  ret = i18n_ulocale_get_default(&defualt_locale);
+  if (ret != I18N_ERROR_NONE) {
+    FT_LOGE("i18n_ulocale_get_default() failed.");
+    return;
+  }
+
+  std::string without_encoding_type(defualt_locale);
+  auto n = without_encoding_type.find('.');
+  without_encoding_type.erase(n, without_encoding_type.length() - n);
+
+  flutter_locale = GetFlutterLocale(without_encoding_type.data());
+  if (flutter_locale) {
+    FT_LOGD("Choose Default locale[%s]", without_encoding_type.data());
+    flutter_locales.push_back(flutter_locale);
+  }
+
   int32_t count = i18n_ulocale_count_available();
   if (count > 0) {
```

Comment:
Don't you need to remove this condition?

Reply:
I fixed it